setup.py

```diff
@@ -169,6 +169,13 @@ def do_setup(package_data):
             "torch",
             "tqdm",
             "typing_extensions",
+            "einops",
+            "webdataset==0.1.103",
+            "matplotlib",
+            "pytorchvideo==0.1.5",
+            "wandb",
```

Comment:
TODO [for Susan]: Installation of wandb also breaks tests - either fix text, or fix how wandb is instantiated.

Reply:
Split out in https://github.com/facebookresearch/metaseq/pull/677